src/csharp/Extensions/Microsoft.Spark.Extensions.Delta.E2ETest/DeltaFixture.cs

```diff
@@ -48,7 +48,7 @@ public DeltaFixture()
 
             Environment.SetEnvironmentVariable(
                 SparkFixture.EnvironmentVariableNames.ExtraSparkSubmitArgs,
-                $"--packages io.delta:{deltaVersion} {confStr}");
+                $"{confStr} --packages io.delta:{deltaVersion}");
```

Comment:
revert change.

Reply:
Done

Reply:
You'll want to modify how you are adding the extra packages in the AddPackages method.

Reply:
Done, thanks!